util/fipstools/acvp/acvptool/subprocess/kdf-components.go

```diff
@@ -0,0 +1,349 @@
+// Copyright (c) 2020, Google Inc.
```

Comment:
NP: can you make 2 reviews (in any order):
1. Move/rename files with no changes
2. Update logic

As is it's hard to tell what you changed and we would have to review the entire file as if it is a new file

Reply:
Ok, I'll make sure to do that in the future. Did you want me to close this PR and resubmit it as well?

Reply:
Yes please, I'd prefer not to review the whole 349 line file.